packages/build-tools/src/utils.ts

```diff
@@ -16,6 +16,12 @@ export async function calculateChecksum(filePath: string) {
   }
 }
 
+export function calculateCheckSum(data: string | Buffer): string {
```

Comment:
Let's keep the spelling the same for both functions:
```suggestion
export function calculateChecksum(data: string | Buffer): string {
```